pkg/azurefile/azurefile_options.go

```diff
@@ -67,6 +68,7 @@ func (o *DriverOptions) AddFlags() *flag.FlagSet {
 	fs.BoolVar(&o.EnableVHDDiskFeature, "enable-vhd", true, "enable VHD disk feature (experimental)")
 	fs.BoolVar(&o.EnableVolumeMountGroup, "enable-volume-mount-group", true, "indicates whether enabling VOLUME_MOUNT_GROUP")
 	fs.BoolVar(&o.EnableGetVolumeStats, "enable-get-volume-stats", true, "allow GET_VOLUME_STATS on agent node")
+	fs.BoolVar(&o.EnableKataCCMount, "enable-kata-cc-mount", false, "enable Kata Confidential Containers mount")
```

Comment:
pls set as true  in this PR for testing in the beginning, thus we could make sure it could pass e2e tests.
when this PR is ready for merge, we could create a helm chart flag, set as true in e2e tests and false by default in the end.